konflux/cpu-ubi9.conf

```diff
@@ -0,0 +1,6 @@
+BASE_IMAGE=quay.io/aipcc/base-images/cpu:3.0-1757673655
+LLAMA_STACK_VERSION=0.0
+WHEEL_RELEASE_PROJECT_ID=71275045
+WHEEL_RELEASE_PACKAGE=llama-stack-wheels
+WHEEL_RELEASE_AARCH64=0.0.272+llama-stack-cpu-ubi9-aarch64
+WHEEL_RELEASE_X86_64=0.0.272+llama-stack-cpu-ubi9-x86_64
```

Comment:
@riprasad So these 2 lines would need a manual update for every https://gitlab.com/redhat/rhel-ai/llama-stack/pipeline/-/releases ?

Reply:
Until automated updates are configured. @moulalis is working on that